recipes/pydusa/meta.yaml

```diff
@@ -3,8 +3,8 @@ package:
     version: 1.15
 
 source:
-    url: https://github.com/cryoem/pydusa/archive/master.tar.gz
-    fn: pydusa-master.tar.gz
+    url: https://github.com/cryoem/pydusa/archive/{{GIT_PYDUSA_BRANCH}}.tar.gz
```

Comment:
I suspect `GIT_PYDUSA_BRANCH` won't be inherited by `conda-build`. It may have to be listed under build/script_env. See docs, https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html#inherited-environment-variables.

Reply:
This still needs to be wrapped in `environ.get()`, doesn't it?

`{{ environ.get(GIT_PYDUSA_BRANCH) }}`

Reply:
According to the documenatation 

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#templating-with-jinja

One only needs environ if one want to set a default value if the variable is empty.